packages/fxa-auth-server/lib/server.js

```diff
@@ -202,9 +203,9 @@ async function create(log, error, config, routes, db, translator, statsd) {
     defineLazyGetter(request.app, 'acceptLanguage', () =>
       trimLocale(request.headers['accept-language'])
     );
-    defineLazyGetter(request.app, 'locale', () =>
-      translator.getLocale(request.app.acceptLanguage)
-    );
+    defineLazyGetter(request.app, 'locale', () => {
```

Comment:
Do you prefer an explicit return? (I don't mind, just wondering 🙂)

Reply:
I do 🙄 , but I like consistency better, so I think I'll change it to the short hand form.